index.html

```diff
@@ -11,8 +11,10 @@
     <h1>The Black Binder</h1>
     <h2>Your number one directory for Black Culture</h2>
     <p>Enter your email address and we will let you know when we are almost ready to go live</p>
-    <input type="text" name="Email" value="Your email address...">
-    <button type="button">Notify Me</button>
+    <form method="POST" action="http://formspree.io/hello@theblackbinder.com">
+        <input type="email" name="email" placeholder="Your email...">
```

Comment:
Can you change the placeholder to an example of an email address? E.g. `hello@example.com`